onadata/apps/api/viewsets/xform_viewset.py

```diff
@@ -710,8 +710,11 @@ def delete_async(self, request, *args, **kwargs):
         if request.method == 'DELETE':
             xform = self.get_object()
             resp = {
-                u'job_uuid': tasks.delete_xform_async.delay(xform.pk).task_id,
-                u'time_async_triggered': datetime.now()}
+                u'job_uuid': tasks.delete_xform_async.delay(
+                    xform.pk,
+                    request.user.id).task_id,
+                u'time_async_triggered': datetime.now(),
+                u'triggered_by': request.user.username}
```

Comment:
Do we need `triggered_by` since we'll be saving the user who triggered the delete?

Reply:
Mostly kept it they're for information purposes (if that makes sense) wanted to be extra verbose. On the first async task status just in case.

TL;DR: We don't really need it

Reply:
Cool, let's remove it for now and may be add it when we need it

Reply:
I've removed the `triggered_by` field in the latest commits. Please have a look @ivermac